# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1466111862 14400
#      Thu Jun 16 17:17:42 2016 -0400
# Node ID b0e5173521b95f14e4315f41e4eec120ddf27d19
# Parent  efe9df9a1071569d6e93a4cf1fd92e25373c691d
* unistd-wrappers.c: Include stdio.h for ctermid decl.

diff --git a/liboctave/wrappers/unistd-wrappers.c b/liboctave/wrappers/unistd-wrappers.c
--- a/liboctave/wrappers/unistd-wrappers.c
+++ b/liboctave/wrappers/unistd-wrappers.c
@@ -24,16 +24,20 @@ along with Octave; see the file COPYING.
 // headers directly in Octave's C++ source files to avoid problems that
 // may be caused by the way that gnulib overrides standard library
 // functions.
 
 #if defined (HAVE_CONFIG_H)
 #  include "config.h"
 #endif
 
+#include <stdio.h>
+
+#include <stdio.h>
+
 #include <sys/types.h>
 #include <unistd.h>
 
 #include "unistd-wrappers.h"
 
 int
 octave_access_f_ok (void)
 {
